src/icons/index.js

```diff
@@ -36,10 +38,13 @@ export * from './oParlSearch';
 export * from './ownLocation';
 export * from './pen';
 export * from './phone';
+export * from './rain';
 export * from './routePlanner';
 export * from './send';
 export * from './service';
 export * from './share';
+export * from './sunDown';
+export * from './sunUp';
```

Comment:
can you please have a look at Tabler icons to see if they have equivalents, because the SVGs currently used could be under copyright by Apple, as someone said?

Reply:
Ich habe einige Änderungen an der heruntergeladenen svg-Datei vorgenommen und sie so gestaltet. Ich denke, sie ist jetzt im besten Zustand.

<img width="141" alt="image" src="https://github.com/user-attachments/assets/f3461e73-1ff4-4d77-9331-cb763ca81048">